blog/electron-38-0.md

```diff
@@ -0,0 +1,103 @@
+- Fixed an issue where `dialog.showMessageDialog` showed a window incorrectly centered to monitor instead of parent window when passed. [#48215](https://github.com/electron/electron/pull/48215)
+- Fixed an issue where users on MacOS were unable to interact with a webpage loaded via loadURL. [#47575](https://github.com/electron/electron/pull/47575)
+
+### Breaking Changes
+
+### Removed: macOS 11 support
+
+macOS 11 (Big Sur) is no longer supported by [Chromium](https://chromium-review.googlesource.com/c/chromium/src/+/6594615).
+
+Older versions of Electron will continue to run on Big Sur, but macOS 12 (Monterey)
+or later will be required to run Electron v38.0.0 and higher.
+
+### Removed: `ELECTRON_OZONE_PLATFORM_HINT` environment variable
+
+The default value of the `--ozone-plaftform` flag [changed to `auto`](https://chromium-review.googlesource.com/c/chromium/src/+/6775426).
+
+You should use the `XDG_SESSION_TYPE=wayland` environment variable instead to use Wayland.
+
+### Removed: `plugin-crashed` event
+
+The `plugin-crashed` event has been removed from `webContents`.
+
+### Deprecated: `webFrame.routingId` property
+
+The `routingId` property will be removed from `webFrame` objects.
+
+You should use `webFrame.frameToken` instead.
+
+### Deprecated: `webFrame.findFrameByRoutingId(routingId)`
+
+The `webFrame.findFrameByRoutingId(routingId)` function will be removed.
+
+You should use `webFrame.findFrameByToken(frameToken)` instead.
+
+## Google Summer of Code Concludes
+
+Our two [Google Summer of Code](https://summerofcode.withgoogle.com/) contributors have just completed their summer projects!
+
+- [@nilayarya](https://github.com/nilayarya) crafted a new [Save/Restore Window State API](https://github.com/electron/rfcs/pull/16/) in Electron core. The new APIs will provide a built-in, standardized way
+  to handle window state persistence. See Nilay's original RFC at [electron/rfcs#16](https://github.com/electron/rfcs/pull/16).
+- [@hitarth-gg](https://github.com/hitarth-gg) put a lot of hard work into modernizing the long-dormant [Devtron](https://github.com/electron-userland/devtron) extension using Chrome Manifest V3 APIs.
+  This project provides tooling for developers to debug IPC communication, track event listeners, and visualize module dependencies in their Electron applications.
+
+Stay tuned for a more detailed blog post outlining their projects and the outcomes.
+
+## End of Support for 35.x.y
+
+Electron 35.x.y has reached end-of-support as per the project's [support policy](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+| E38 (Aug'25) | E39 (Oct'25) | E40 (Jan'26) |
```

Comment:
```suggestion
| E38 (Sep'25) | E39 (Oct'25) | E40 (Jan'26) |
```
If this is indicating the month of each versions first stable release, then 38 should be in September